src/renderer/components/+extensions/installed-extensions.tsx

```diff
@@ -102,29 +106,34 @@ export const InstalledExtensions = observer(({ extensions, uninstall, enable, di
           ),
           version,
           status: (
-            <div className={cssNames({[styles.enabled]: getStatus(isEnabled) == "Enabled"})}>
-              {getStatus(isEnabled)}
+            <div className={cssNames({[styles.enabled]: isEnabled, [styles.invalid]: !isCompatible})}>
+              {getStatus(extension)}
             </div>
           ),
           actions: (
             <MenuActions usePortal toolbar={false}>
-              {isEnabled ? (
-                <MenuItem
-                  disabled={isUninstalling}
-                  onClick={() => disable(id)}
-                >
-                  <Icon material="unpublished"/>
-                  <span className="title" aria-disabled={isUninstalling}>Disable</span>
-                </MenuItem>
-              ) : (
-                <MenuItem
-                  disabled={isUninstalling}
-                  onClick={() => enable(id)}
-                >
-                  <Icon material="check_circle"/>
-                  <span className="title" aria-disabled={isUninstalling}>Enable</span>
-                </MenuItem>
+              { isCompatible && (
```

Comment:
Might be nice to one day have an "Install latest" or "find compatible" option when the the extension is not compatible, but that really means knowing if there is a compatible version available...

Reply:
There has been discussions about moving extensions to catalog. Catalog could show list of all known extensions with actions to install/uninstall/update.